servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpResponseStatus.java

```diff
@@ -606,7 +605,7 @@ private static HttpResponseStatus valueOf(final int statusCode) {
             case 424:
                 return FAILED_DEPENDENCY;
             case 425:
-                return UNORDERED_COLLECTION;
+                return TOO_EARLY;
```

Comment:
By some reason GH shows these changes (introduced by @karim-elngr) as part of your PR even after you merged the latest master into this branch. Could you please rebase and make sure that only your changes are visible on GH for this PR? May be squash will help.

Reply:
Done! I squashed all my commits and rebased  again